### ISSUE TITLE
Add support for accessibility

### ISSUE THREAD

**Issue by defagos** [SRGSSR/srgletterbox-apple#7]
Disabled users should be able to use Letterbox easily.

### Issue overview

* [ ] Incorrect behavior
* [X] Feature request
* [ ] Crash
* [ ] Help / documentation
* [ ] Other: _classification_

### Description of the problem

Disabled users must be able to use the Letterbox player. Controls should be made always visible for easy navigation between buttons.

In particular, the behavior should be good when the system accessibility floating button is used.
